### DISCUSSION TITLE
Pledges Redux

### DISCUSSION THREAD

**Discussion by BenJam** [opencollective/opencollective#6027] [Ideas]
Pledges were launched in [October 2018](https://github.com/opencollective/opencollective/issues/1168):

> A Pledge can be thought of as a pending Order for a Collective that does not yet exist. A user or organization can create a pledge of some monetary amount for an open source project or a meetup, and that Pledge will show on the user or org collective page. Once the Pledge is created, we can encourage the user / org to reach out to the project or meetup and point to that Pledge. Once the collective is created for that project, the pledge can be claimed and fulfilled by the pledge creator. When pledges have been made for a new, inactive collective, then that collective can be "claimed" by one of the admins for the github repo linked to that collective.

Some updates followed in 2019 but, by 2020 the [discussion about axing pledges](https://github.com/opencollective/opencollective/issues/3135) had bugun:

>Currently pledged collectives: 65
Total amount currently pledged: $6,283
Collectives claimed: 9 (techx,ui-avatars,hyperapp,we-cropper,exercism,octobercms,sentinel,prettier,thetorproject)
Amount donated through honored pledges: not enough data to know

Two years later those stats [look like this](https://opencollective-metabase.herokuapp.com/dashboard/117-pledges): 

- 639 pledges by 352 users to 118 Collectives
- 124k currently pledged

around 40k of pledges are pledges to a parallel experiment in which pledges were [used to drive feature prioritisation](https://opencollective.com/html-inert-in-webkit-safari). It's thought that these pledges were never intended to be fulfilled and, instead were treated as votes. 

So, four years later, we _could_ admit that pledges are, ultiamtely, a failure. But we should at least look at some of the issues. 

## Issues

Our first issue is actually presented in the review back in 2020:

> The maintainer claimed the Collective, and then was confused/upset that the promised money wasn't there. I had to explain to them that we never collected any credit card numbers, and we just notified anyone who pledged to complete their payment but can't force them.

The current flow does not commit funds from an individual or organisation only signals to the potential maintainer that there are some users and organisations may commit if the collective was claimed. Sadly we dont have any data to reliably confirm how many pledges 'converted' like this but, given anecdotal exprience, I think it's low. 

Other issues might exist, isssue-sleuthing reveals that, perhaps the [user experience](https://github.com/opencollective/opencollective/issues/1389) wasn't the best or that [BackYourStack](https://github.com/opencollective/opencollective/issues/1129) didn't take off as planned. (there's another thread on that later). For me this lack of commitment, and the inability for us to drive users through this process is the problem. 

## Why we still need pledges

Open Collective, and specifically Open Source Collective is (happily) part of a growing landscape of solutions for projects looking to financially sustian their work. Unfortuantely that landscape is fractured, which impacts our ability to offer compellingly comprehensive solutions to problems (like BackYourStack's core use case) and to empower communities to demand better of open source maintainers, encouraging them to adopt [open, transparent and inclusive approaches](https://opencollective.com/vuejs) to using money as a community over funding a [single individual mainatianer](https://github.com/sponsors/yyx990803). Want to support all the community fridge projects in PNW Pledge to support them and encourage others to do the same? Wish your favourite open source project paid for experienced tech writers to improve the documentation? Pledge to the project and tell them!

## Opportunties

I believe we have the opportunity to solve the core problem with Pledges today, Funds. 

Funds are a mechanism for us to commit users to their pledges. Google, Salesforce, AirBnB and Indeed's organisation profiles _are_ funds. They have a committed balance held with OSC on account that gives them the flexiblity to donate as and when they want. We could use the same tools to ask individuals (and companies) to commit those funds to pledges. We can aso provide the flexibility for those entities to change their pledged structure as they see fit, but this is a second order issue. 

## An alternative

If we dont like the idea of creating individual funds to hold Pledges, there is another route: _Campaigns_. 

Campaigns are like portfolios of projects. They can include Collectives and Pledged Collectives. They encourage curation according to territory or cause and they allow us to commit pledges (the campagin is effectively a Fund) through that fund. Campaigns could help us solve some of the fractulisation issues — campagins to support the top 100 Ruby, Rust, JavaScript project could exist alongside campagins to support all the community kitchens in Chicago — but they also provide a route for us to enagage and provide a real role to our fans to help us support causes they care about _and_ grow the number of projects on the platform. 

## Details Details

I'm sharing none. I just leave this for others as a prompt for more conversation. Do these ideas excite you? If it fails how does it fail? If it was going to work what would we need in place. These are the questions that come for those who say yes.

**Comment by Betree** [comment]
Sorry for jumping on the conversation so late. I believe it's an important one to have as pledges are currently in this half-baked state that is not satisfying, yet as you said it's still an interesting feature that can help with some projects like BackYourStack.

There are still difficult questions to resolve, I'm not sure for example that we are legally allowed to collect money for a project that's not registered (that was presented as a blocker at the time). We also need a refund strategy for projects who don't want to join, we had cases like that in the past (on the top of my head, Peek (screencast recorder) and Clementine (music player)).

**Comment by natehn** [comment]
> The current flow does not commit funds from an individual or organisation only signals to the potential maintainer that there are some users and organisations may commit if the collective was claimed.

This feels valuable to me, even if many of the pledges are not converted. It signals to the maintainer that they could get support. And perhaps we can do something like gather info from the donors (payment info?) so that all they have to do is press a button to confirm they want to act on their pledge.

> We could use the same tools to ask individuals (and companies) to commit those funds to pledges.

Love this idea for companies, but I'm skeptical of Funds being useful for most individuals. Individuals tend to have a lot less expendable income that they're willing to see go unused. My experience in fundraising with donations from individuals is that they are slower to confirm a pledge than they are on a spur-of-the-moment donation to an active project. Organizations, on the other hand, are far more strategic.

Overall, I like Pledges as an idea. I think they may have been killed off too early to know exactly how valuable they are. Even if they only land for companies/Funds, I think it could be worth the effort.

**Comment by natehn** [comment]
@Betree my understanding is that money is not collected when someone makes a pledge, so we will not run into those sorts of issues. @BenJam am I correct?

**Reply by Betree** [reply]
@natehn Yes, that's how it works today; but @BenJam's proposal above is to actually collect the money via funds (at least that's my understanding)

**Reply by natehn** [reply]
Ah! Well, my thought is that the money would remain in the fund, to be used elsewhere. As in, people could put money in the fund, and then make a pledge from that fund, but if the pledge is never called upon or is rejected, then the funds stay in their fund with OSC.

**Reply by Betree** [reply]
Then what happens if I clearly say I don't want my project to be on Open Collective, and explicitly ask to remove the pledges? What will become of the funds, will they be refunded (knowing that payment providers usually don't let you refund charges after a certain amount of time and that they may still take a fee)? Or allocated to other projects (which, as a financial contributor, could be perceived as a betrayal)?

**Reply by natehn** [reply]
I'm thinking that they would stay in a Fund...I think we would ask people to put money in the fund first, and then allow them to use the money in the fund to make a pledge - two separate decision points. That way, we would not need to refund them
